server/views/formPages/recat/decision.html

```diff
@@ -80,59 +79,45 @@ <h1 id="securityInputHeader" class="govuk-heading-l">{{ pageTitle }}</h1>
     text: "Consider them for YOI (open)",
     label: { classes: "govuk-!-font-weight-bold" }
   } %}
+  {% set optionR =  {
+    value: "R",
+    id: "closedOption",
+    checked: data.recat.decision.category == 'R',
+    hint: {
+      html: hintText('closed')
+    },
+    text: "Closed",
+    label: { classes: "govuk-!-font-weight-bold" }
+  } %}
+  {% set optionT =  {
+    value: "T",
+    id: "openOption",
+    checked: data.recat.decision.category == 'T',
+    hint: {
+      html: hintTextOpen('open')
+    },
+    text: "Consider them for open",
+    label: { classes: "govuk-!-font-weight-bold" }
+   } %}
 
-  {% set itemList = [ optionB, optionC, optionD, optionI, optionJ ] if data.details.youngOffender else [ optionB, optionC, optionD ] %}
-{% endif %}
+  {% if user.activeCaseLoad.female %}
+    {% set itemList = [ optionI, optionJ, optionR, optionT ] if data.details.youngOffender else [ optionR, optionT ] %}
+  {% else %}
+    {% set itemList = [ optionB, optionC, optionD, optionI, optionJ ] if data.details.youngOffender else [ optionB, optionC, optionD ] %}
+  {% endif %}
 
-{% if user.activeCaseLoad.female %}
   {{ govukRadios({
     name: "category",
     classes: "govuk-radios",
     fieldset: {
       legend: {
-        text: "Which category is most suitable for this person?",
-        isPageHeading: false
```

Comment:
Reminder to check

Reply:
In prototype: `What security category is most suitable for this prisoner?`
Check config/recat.js and recat/DecisionSpecification too

Reply:
Just checked the git branches - on main it states "What category is most suitable for this person?"  On the prototype for women's estate option 2 it states "What category is most suitable for this person?"  at https://categorisation-prototype-main.apps.live.cloud-platform.service.justice.gov.uk/v1/cat/decision?cya=false.  I am aware that Lisa mentioned some content rules where you can't call an offender "prisoner" 
I will have to see if I can change the text on whether they are male or female.

Reply:
I'd suggest to discuss it with Barry